alephium/contracts/governance.ral

```diff
@@ -30,34 +42,32 @@ TxContract Governance(
 
     // return emitterChainId, targetChainId, emitterAddress, sequence, payload
     fn parseBody(body: ByteVec) -> (U256, U256, ByteVec, U256, ByteVec) {
-        let timestamp = u256From4Byte!(byteVecSlice!(body, 0, 4))
-        let nonce = u256From4Byte!(byteVecSlice!(body, 4, 8))
+        // skip the first 8 bytes(timestamp and nonce)
         let emitterChainId = u256From2Byte!(byteVecSlice!(body, 8, 10))
         let targetChainId = u256From2Byte!(byteVecSlice!(body, 10, 12))
         let emitterAddress = byteVecSlice!(body, 12, 44)
         let sequence = u256From8Byte!(byteVecSlice!(body, 44, 52))
-        let consistencyLevel = u256From1Byte!(byteVecSlice!(body, 52, 53))
+        // skip 1 byte of consistencyLevel
```

Comment:
Yes, ETH set the `consistencyLevel` to 15 [in the contract](https://github.com/certusone/wormhole/blob/c2db1116293ab81ddad4e87fc9af175656ab40e1/ethereum/contracts/bridge/BridgeImplementation.sol#L24), I think we should check `consistencyLevel >= minConfirmations' in the contract to avoid attacks.